src/shared/store/reducers/modals.ts

```diff
@@ -9,11 +9,13 @@ export const DEFAULT_MODALS_STATE = {
   createOrganizationModel: false,
   createProjectModel: false,
   createStudioModel: false,
+  aboutModel: false,
 };
 export type modalsState = {
   createOrganizationModel: boolean;
   createProjectModel: boolean;
   createStudioModel: boolean;
+  aboutModel: boolean;
```

Comment:
I would suggest renaming this variable to `hasAboutModel` or `isAboutModalOpen` or something that signals that the variable contains a boolean property. Right now it sounds like it holds the About Modal component.